### ISSUE TITLE
Ability to specify Java command-line arguments such as 'RELATIVE_INCLUDE'

### ISSUE THREAD

**Issue by hannotify** [esteinberg/plantuml4idea#336]
Hi! First of all: thank you for this amazing IntelliJ plugin. My team uses it currently to create [C4](//c4model.com) architecture diagrams using PlantUML.

Now I cloned the [C4-PlantUML](https://github.com/plantuml-stdlib/C4-PlantUML) repository and want to use its files locally because of a company proxy. When creating a component diagram I include the local copy of [C4_Component.puml](https://github.com/plantuml-stdlib/C4-PlantUML/blob/master/C4_Component.puml), for example. But that file includes the `C4_container.puml` from the internet by default. If I want to use a local copy, I have to set the `RELATIVE_INCLUDE` command-line argument. Is there a way to achieve this with the plugin?

I have noticed there is a settings field for `PLANTUML_LIMIT_SIZE`; we could perhaps generalize this and allow custom command-line arguments instead? I would be more than happy to contribute to that.

**TL;DR:** 
Can we specify Java command-line arguments such as `RELATIVE_INCLUDE` in the plugin. Or would you accept contributions to achieve this?

**Comment by krasa** [comment]
It can be done like this:
![image](https://user-images.githubusercontent.com/1160875/142648858-bbde6613-7568-4538-bf5e-b592374da945.png)

Perhaps a specialized variable table would be better.

**Comment by gebezs** [comment]
Modifying the IDE's VM options sounds overkill. I'm not familiar how Idea's plugin system works but after a quick search I found this: https://github.com/esteinberg/plantuml4idea/blob/master/adapter/src/org/plantuml/idea/adapter/Utils.java#L93

Am I right to assume that
* The `Utils.java` needs an update to handle the dropdown and/or a arbitrary list of system properties.
* The settings page needs a dropdown and/or the list.

I couldn't find the test for that. Can you please point me to the right direction?

**Comment by krasa** [comment]
Yes, Utils.java could handle PLANTUML_SECURITY_PROFILE as well. Until then, IDE's VM options is the way to do it.

There is no test :-)
